core/field_multilineinput.js

```diff
@@ -15,18 +15,18 @@
 goog.module('Blockly.FieldMultilineInput');
 goog.module.declareLegacyNamespace();
 
-goog.require('Blockly.Css');
-goog.require('Blockly.Field');
-goog.require('Blockly.fieldRegistry');
-goog.require('Blockly.FieldTextInput');
-goog.require('Blockly.utils');
-goog.require('Blockly.utils.aria');
-goog.require('Blockly.utils.dom');
-goog.require('Blockly.utils.KeyCodes');
-goog.require('Blockly.utils.object');
-goog.require('Blockly.utils.Svg');
-goog.require('Blockly.utils.userAgent');
-goog.require('Blockly.WidgetDiv');
+const Css = goog.require('Blockly.Css');
+const Field = goog.require('Blockly.Field');
+const fieldRegistry = goog.require('Blockly.fieldRegistry');
+const FieldTextInput = goog.require('Blockly.FieldTextInput');
+const KeyCodes = goog.require('Blockly.utils.KeyCodes');
+const {addClass, createSvgElement, getFastTextWidth, getTextWidth, removeClass} = goog.require('Blockly.utils.dom');
+const {DIV} = goog.require('Blockly.WidgetDiv');
```

Comment:
I'm not sure how I feel about this one. Assigning `const div = DIV` is a bit confusing, especially because of the terrible constant-syntax naming of `DIV`.